tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -37,7 +37,7 @@ def setup(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo):
 
     hwsku = duthost.facts['hwsku']
     minigraph_facts = duthost.get_extended_minigraph_facts(tbinfo)
-    port_alias_facts = duthost.port_alias(hwsku=hwsku, include_internal=False)['ansible_facts']
+    port_alias_facts = duthost.port_alias(hwsku=hwsku, include_internal=True)['ansible_facts']
```

Comment:
What internal ports will be included? Inband ports?  Also the change is not only for voq system, right?

Reply:
why do we need this change ? The port alias is not needed for inband ports I think we can ignore them.
The port alias is mainly needed for front panel ports.